webapi/src/main/scala/org/knora/webapi/messages/admin/responder/usersmessages/ProjectPayloadsADM.scala

```diff
@@ -0,0 +1,45 @@
+package org.knora.webapi.messages.admin.responder.usersmessages
+
+import org.knora.webapi.IRI
+
+/**
+ * Project payload
+ */
+sealed abstract case class ProjectCreatePayloadADM private (
```

Comment:
strictly speaking this does not need a smart constructor, since it doesn't do any "smart" checks on it and could be written as a `final case class ProjectCreatePayloadADM(...)`. We can leave it like it is and if no spart checks get added over time, then we can refactor it.